wagtail/snippets/views/snippets.py

```diff
@@ -501,16 +530,37 @@ def history_label(self):
             "model_name": self.model._meta.verbose_name
         }
 
+    def get_template_names(self):
+        return self.viewset.get_templates(
+            "revisions_compare",
+            fallback="wagtailadmin/generic/revisions/compare.html",
```

Comment:
Just realised this should be `self.template_name` for consistency with the others. Happy to fix this, but I'll hold on in case your review is in progress @gasman.

Reply:
Not reviewing right now - go ahead!

Reply:
Thanks! Should be all good to review now.